### PR TITLE
Fix a typo in logging message & redundant f-string usage

### PR DESCRIPTION
**Pull request by peterjc**
Noticed while investigating #353 for which I intend to add a little more logging.

## Type of change

- [X] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [ ] This is a documentation update

## Action Checklist

- [X] Work on a single issue/concept (if there are multiple separate issues to address, please use a separate pull request for each)
- [X] Fork the `pyani` repository under your own account (please [allow write access for repository maintainers](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
- [X] Set up an appropriate development environment (please see `CONTRIBUTING.md`)
- [X] Create a new branch with a short, descriptive name
- [X] Work on this branch
  - [X] style guidelines have been followed
  - [ ] new code is commented, especially in hard-to-understand areas
  - [ ] corresponding changes to documentation have been made
  - [ ] tests for the change have been added that demonstrate the fix or feature works
- [ ] Test locally with `pytest -v` **non-passing code will not be merged**
- [X] Rebase against `origin/master`
- [X] Check changes with `flake8` and `black` before submission
- [X] Commit branch
- [X] Submit pull request, describing the content and intent of the pull request
- [ ] Request a code review
- [ ] Continue the discussion at [`Pull requests` section](https://github.com/widdowquinn/pyani/pulls) in the `pyani` repository

### CODE REVIEW

**Comment by peterjc** [comment]
Interesting - CodeFactor flagged one of these changes was in unreachable code following ``raise NotImplementedError``

**Comment by baileythegreen** [comment]
@peterjc CodeFactor is flagging the change in `anib` because that subcommand hasn't been implemented yet in `master`. There is an `anib_123` branch where it is. We won't be concerned with the warning here.

**Review by baileythegreen — CHANGES_REQUESTED** [review]
I have left some changes I would like to see (in line with style cleaning changes I've been working on)  in comments. Once the changes are made, this can be merged.

**Comment by widdowquinn** [comment]
Sorry to jump in here, but AFAIAA we should not use f-strings in `logging` calls. There is more info at https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles but essentially string formatting is deferred until it cannot be avoided, and f-strings format early, as explained here: https://klichx.dev/2019/11/09/f-strings-and-python-logging/

**Review by widdowquinn — CHANGES_REQUESTED** [review]
Please don't use f-strings for logger messages.

**Comment by baileythegreen** [comment]
@widdowquinn Okay, I didn't realise that was a considered choice for optimisation of the logging. I'll go back and look at where I might have changed logging strings and fix them.

@peterjc Sorry for the confusion. If you can revert the last commit, then I can merge it.

**Comment by widdowquinn** [comment]
> @widdowquinn Okay, I didn't realise that was a considered choice for optimisation of the logging. I'll go back and look at where I might have changed logging strings and fix them.

It isn't just optimisation either - some tools (e.g. monitoring tools like Sentry) catch the log strings - for instance to group messages on the basis of identical patterns. When messages get concretised with values _before_ the call to `logger` this isn't possible.

**Comment by peterjc** [comment]
Reverted. Interesting reading, thank you.

File: pyani/scripts/subcommands/subcmd_anib.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def subcmd_anib(args: Namespace) -> None:
         "Compiling pairwise comparisons (this can take time for large datasets)..."
     )
     comparisons = list(permutations(tqdm(genomes, disable=args.disable_tqdm), 2))
-    logger.info(f"\t...total parwise comparisons to be performed: {len(comparisons)}")
+    logger.info(f"\t...total pairwise comparisons to be performed: {len(comparisons)}")
```
Comment (baileythegreen) [inline pyani/scripts/subcommands/subcmd_anib.py:185]:
```suggestion
    logger.info("\t...total pairwise comparisons to be performed: %s", len(comparisons))
```

**Pull request merged**